docs/sources/tutorials/get-started.md

```diff
@@ -14,21 +14,40 @@ You don't need to know about the {{< param "PRODUCT_NAME" >}} [configuration syn
 
 ## Prerequisites
 
-The tutorials require a Linux or Unix environment with Docker installed.
+This first tutorial requires a Linux, Unix, or Mac environment with Docker installed.
 The examples run on a single host so that you can run them on your laptop or in a Virtual Machine.
 You are encouraged to try the examples using a `config.alloy` file and experiment with the examples yourself.
 
-To run the examples, you should have an {{< param "PRODUCT_NAME" >}} binary available. You can follow the instructions on how to [Install {{< param "PRODUCT_NAME" >}} as a Standalone Binary][install] to get a binary.
+## Install {{< param "PRODUCT_NAME" >}} and start the service
+
+### Linux
+
+Follow the instructions on the [Linux Install] page for the steps for several popular
+Linux distributions.  
+
+{{< admonition type="tip" >}}
+Make sure to follow the optional install step to enable the UI, we will be referring to it in this tutorial.
+{{< /admonition >}}
+
+Once you have completed this, follow the instructions to [Run on Linux] using `systemctl`.
```

Comment:
```suggestion
Once you have completed this, follow the instructions to [Run on Linux][] using `systemctl`.
```

Syntactically not required, but we should include the second `[]` fro consistency.  Same applies throughout the rest of the topic.